src/edu/ucsb/cs56/projects/games/fish_animation/Menu.java

```diff
@@ -559,6 +547,7 @@ public void HowToPlay() {
 		
 		textLabel.setText("<html><h3>Welcome to Fish Animation !</h3>"
 				+ "<p>Hello! The goal of the game is to eat as many fish as you can. "
+                + "Right now, the target point is 50. Whenever you get 50 or more points, you win! "
```

Comment:
The first part should be "Right now, the target is 50 points."  Fixing the indentation would be nice too.